mne/viz/_3d.py

```diff
@@ -382,18 +426,13 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
     hemi : str, 'lh' | 'rh' | 'split' | 'both'
         The hemisphere to display. Using 'both' or 'split' requires
         PySurfer version 0.4 or above.
-    colormap : str
-        The type of colormap to use.
+    colormap : str | array
+        Name of colormap to use or a custom look up table (an n x 4
```

Comment:
It's the number of interpolation points in the colormap. I'll add that into the docstring.